### PR TITLE
Improve Task Hunt by using doUGL for Flares

### PR DESCRIPTION
**Pull request by jokoho48**
### When merged this pull request will:

1. *Add the use of doUGL for Flares in taskHunt*
2. *Add Parameter for using Flares for Task Hunt*
3. *Simplify doUGL by not Prefiltering Units that are local or playersDescribe what this pull request will do*

### CODE REVIEW

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -516,6 +516,12 @@
             <Polish>Obszar na którym grupa będzie szukać przeciwników.\nDomyślnie moduł wyszukuje tylko wrogich graczy.</Polish>
             <Russian>Расстояние на котором группа охоты будет искать врагов.\nПо умолчанию этот модуль нацелен только на вражеских игроков.</Russian>
         </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_TryUGLFlare_DisplayName">
+            <English>Use Flare</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskHunt_TryUGLFlare_Tooltip">
+            <English>Should the Units use Flare on Contact.</English>
```
Comment (nk3nny) [inline addons/wp/stringtable.xml:523]: Sets how units use flares when hunting for enemies.

Comment (jokoho48) [reply addons/wp/stringtable.xml:523]:
```suggestion
            <English>Sets how units use flares when hunting for enemies.</English>
```

File: addons/wp/stringtable.xml [hunk 2]
```diff
@@ -657,6 +663,15 @@
...
             <English>Suppressed</English>
         </Key>
+        <Key ID="STR_Lambs_WP_Enabled">
+            <English>Enabled</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Disabled">
+            <English>Disabled</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_OnlyIfUGL">
+            <English>Only if Units have UGL Flares</English>
```
Comment (nk3nny) [inline addons/wp/stringtable.xml:673]: Only when units are equipped with an UGL and flares

Comment (jokoho48) [reply addons/wp/stringtable.xml:673]:
```suggestion
            <English>Only when units are equipped with an UGL and flares</English>

```

**Pull request merged**